app/controllers/krikri/reports_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Krikri
+  class ReportsController < ApplicationController
+
```

Comment:
I occasionally got the error your described as well, in which Rails rendered Blacklight's layout instead of Krikri's.  I was never able to pin down a solution b/c the problem often self-corrected when I re-loaded the Rails sever.  I will try adding an explicit call to the Krikri layout as you suggest, and we may need to continue to monitor the problem.